### PR TITLE
created privacy.html, created privacy.css and styled html page to mat…

### PR DESCRIPTION
**Pull request by manni8436**
…ch rest of site

# Hackathon Git Labs Project PR 🎉🎉

👉 _**Please remove the below and replace with your own values, leaving the headers where they are.**_ 👈

## PR contains:
- Created `privacy.html` page.
- Created `privacy.css` page.
- Styled `privacy.html` page to match whole site.

## Hacktoberfest 2022:
- [x] Does this PR follow the `intermediate.md` guide and submitted as part of Hacktoberfest 2022?

_Please note, we cannot accept `basic.md` PR submissions for Hacktoberfest, it must be a custom HTML or CSS creation following the `intermediate.md` guide._

## Breaking changes
None

## Screenshots
image

image

image

### CODE REVIEW

File: assets/css/privacy.css [hunk 1]
```diff
@@ -0,0 +1,37 @@
+@import url('https://fonts.googleapis.com/css2?family=Nunito:wght@300&family=Roboto:wght@300&display=swap');
```
Comment (auxfuse) [inline assets/css/privacy.css:1]: Could we consider merging the common styles in this CSS file into the [primary stylesheet](https://github.com/auxfuse/hackathon-git-labs/blob/main/assets/css/style.css) for the project? Adding an additional stylesheet with minimal styles to it for the one page seems a little bit of file directory and separation of concern overkill.

I would recommend creating a new section in the `style.css` file called "Privacy" just before the Media Queries section on line 629.

Once done, please remove this file from the `assets/css` sub-directory and add/commit/push back to the branch associated with this PR to auto-update the PR. 🤜

File: privacy.html [hunk 2]
```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="assets/css/style.css">
+    <link rel="stylesheet" href="assets/css/privacy.css">
```
Comment (auxfuse) [inline privacy.html:9]: Once the privacy.css file is merged into the primary style.css file, we can remove this line.

**Review by auxfuse — CHANGES_REQUESTED** [review]
Hi @manni8436 see requested changes 😄 

Great work all-in-all my man. Just some small considerations to ensure standardisation of the file directory 🤜 Thanks in advance.

File: assets/css/privacy.css (hunk 1 above)
Comment (manni8436) [reply assets/css/privacy.css:1]: Changes have been made and all added to the `style.css` file. 🤛

File: privacy.html (hunk 2 above)
Comment (manni8436) [reply privacy.html:9]: This has been deleted now 🤛

**Review by auxfuse — APPROVED** [review]
Awesome work @manni8436 Approved! 👏 🚀

**Pull request merged**